.github/workflows/Deploy

```diff
@@ -0,0 +1,65 @@
+name: Deploy
+
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
```

Comment:
v4 is the latest.

Reply:
Yeah, I was going to let dependabot grab those (was going to act as a test that the automerge works as well)